app/pages/learn/01_tutorial/01_your-first-java-app/02_building-with-intellij-idea.md

```diff
@@ -0,0 +1,221 @@
+last_update: 2024-03-12
+---
+
+An IDE (Integrated Development Environment) allows you to quickly build applications, by integrating a source-code editor with the ability to compile and run your code, as well as integration with any other tools you’ll need for software development, including build tools, testing and debugging tools, version control, databases, and so on. And finally, an IDE will let you search and navigate your codebase in ways your file system won’t.
+
+IntelliJ IDEA is the [most widely used IDE for Java developers](https://www.jetbrains.com/lp/devecosystem-2023/java/#java_ide). It has out-of-the-box integrations with all the tools a Java developer will need for software development, with no need to install any additional plugins to get started.
+IntelliJ IDEA is available in two editions:
+- **_IntelliJ IDEA Community Edition_** - free and open-source, licensed under Apache 2.0. It provides all the basic features for Java and Kotlin development.
+- **_IntelliJ IDEA Ultimate_** - commercial, distributed with a 30-day trial period. It provides additional tools and features for web and enterprise development.
+
+## Installing IntelliJ IDEA
+To install IntelliJ IDEA, download the version you want to use from the [IntelliJ IDEA website](https://www.jetbrains.com/idea/) and follow the instructions. For more information on installing IntelliJ IDEA on your OS, see [the documentation](https://www.jetbrains.com/help/idea/installation-guide.html#standalone).
+
+When you launch IntelliJ IDEA for the first time, you’ll see the **Welcome** screen. From here, you create a new project, open an existing project, or get a project from a version control system.
+
+[![Welcome screen](/assets/images/intellij-idea/welcome-screen.png)](/assets/images/intellij-idea/welcome-screen.png)
+
+There is no need to install any additional plugins to get started; the most common integrations are already available as part of the IDE or as bundled plugins. Of course, you can [customize the IDE](https://www.jetbrains.com/help/idea/run-for-the-first-time.html#select-ui-theme) further, or [install additional plugins](https://www.jetbrains.com/help/idea/run-for-the-first-time.html#additional_plugins) if you really want to.
+
+You will need to install a JDK, if you don’t already have one installed. You can take care of that when creating a project.
+
+## Creating a new project
+As mentioned above, we can create a new project from the **Welcome** screen. Alternatively, we can go to **File | New | Project** in the main menu.
+
+[![New Project menu](/assets/images/intellij-idea/new-project-menu.png)](/assets/images/intellij-idea/new-project-menu.png)
+
+In the **New Project** wizard, make sure that **New Project** is selected on the left hand side.
+
+Give your project a name (for example `java-demo`), make sure the selected **Language** is `Java`, and select the **Build system** you prefer. For example, let’s create a Maven project.
```

Comment:
When creating the first project, people will likely not want to use a build tool as they have no idea what that is.

Reply:
Considering there are plenty of questions on SO and other places about which jars to import (and related problems), I'm not sure using build tools can be introduced too soon. But it might be better to have a separate article on them (as well). 
For this article though, we're also adding a testing library in the upcoming testing section - which wouldn't work if we don't add it here.

Reply:
I haven't used IntelliJ lately so I thought as the `IntelliJ` build system option as a plain Java project.Alsso I don't know how much value testing adds at all to an article that's just for getting started with an IDE.